content/modules/flow-ui/pages/creating-components/integrating-into-jmix-ui.adoc

```diff
@@ -0,0 +1,55 @@
+= Integrating into Jmix UI
+
+After creating the client-side and server-side parts of the component, it can be used in xref:views/views.adoc[views] by creating an instance of its Java class and adding it to a parent component.
+
+Usually, the content of a view is defined in an XML file called a descriptor. Taking the Slider component implemented in xref:creating-components/js-component.adoc[] as an example, let's integrate it into the Jmix UI by adding XML support:
+
+. Create the `app-ui-components.xsd` file and define the XML markup for the component:
++
+[source,xml,indent=0]
+----
+include::example$/onboarding/src/main/resources/com/company/onboarding/app-ui-components.xsd[tags=xsd-schema;!theme-toggle-schema]
+----
+
+. Create a component loader that is needed to initialize the component when it is used in the screen XML descriptors.
++
+[source,java,indent=0]
+----
+include::example$/onboarding/src/main/java/com/company/onboarding/component/SliderLoader.java[tags=slider-loader]
+----
+<1> The base class provides helper methods for loading attribute values.
+<2> The `componentLoader()` method returns an instance of the `ComponentLoaderSupport` bean that provides methods for loading common attributes.
+
+. To register the component and its loader with the framework, you need to create a Spring configuration class using the `@Configuration` annotation:
++
+[source,java,indent=0]
+----
+include::example$/onboarding/src/main/java/com/company/onboarding/ComponentRegistrationConfiguration.java[tags=component-registration-configuration;!theme-toggle-registration]
+----
+
+After that, the `Slider` component can be defined in the view's descriptor:
+
+[source,xml,indent=0]
+----
+include::example$/onboarding/src/main/resources/com/company/onboarding/view/customcomponent/jscomponent/js-component-view.xml[tags=slider-xml]
+----
+<1> The `xmlns` attribute specifies the xml namespace for a document. In this case with the `app` prefix.
```

Comment:
"Define a namespace with the same value as in `xmlns` and `targetNamespace` attributes of the component's XSD"